packages/file/src/types/ceph_conf.coffee.md

```diff
@@ -44,13 +44,13 @@ file][ceph-conf] as Ceph daemons expect it.
         'target':
           type: 'string'
           description: """
-          File to write, default to "/etc/pacman.conf".
+          File to write.
           """
       required: ['target']
 
 ## Handler
 
-    handler = ({config, operations: {log}}) ->
+    handler = ({config}) ->
       # log message: "Entering file.types.ceph_conf", level: 'DEBUG', module: 'nikita/lib/file/types/ceph_conf'
       config.target = "#{path.join config.rootdir}" if config.rootdir
```

Comment:
Could you please fix this one, it shall be `#{path.join config.rootdir, config.target}"